groth16-framework/test_data/Groth16VerifierExtensions.sol

```diff
@@ -52,6 +68,8 @@ contract Query2 is Verifier {
         uint256 minBlockNumber;
         uint256 maxBlockNumber;
         bytes32 blockHash;
+        uint256 rewardsRate;
```

Comment:
```suggestion
        uint88 rewardsRate;
```

Reply:
Fixed in commit [d7a6d10](https://github.com/Lagrange-Labs/Euclid-database/pull/21/commits/d7a6d1054fe1853f1d9cf3b8ffa47139efb3792c).